pkg/reconciler/service/kube/kube.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright 2020 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The spacing is not standard for all new copyrights added to this PR. Take a look at other files to see what I mean.